src/imitation/scripts/convert_trajs_in_place.py

```diff
@@ -1,8 +1,7 @@
 """
 This script takes as command-line input multiple paths to pickle files containing
-Sequence[imitation.types.TrajectoryWithRew] or Sequence[imitation.old_types.Trajectory].
-It overwrites each path with a new pickle where the demonstration data is saved as
-Sequence[imitation.types.TrajectoryWithRew].
+Sequence[imitation.types.TrajectoryWithRew]. It overwrites each path with a new pickle
```

Comment:
May want to mention here that this adds `terminal=True`? Otherwise it's not clear what this script does (and in contrast to what this script did previously, adding `terminal=True` might not always be the right thing to do)